mlx_lm/server.py

```diff
@@ -615,9 +619,10 @@ def completion_usage_response(
             "created": self.created,
             "choices": [],
             "usage": {
-                "prompt_tokens": prompt_token_count,
-                "completion_tokens": completion_token_count,
-                "total_tokens": prompt_token_count + completion_token_count,
+                "prompt_tokens": prompt_token_count or 0,
+                "completion_tokens": completion_token_count or 0,
+                "total_tokens": (prompt_token_count or 0)
+                + (completion_token_count or 0),
```

Comment:
There is no need or these to be optional. I would just make them required and simplify the logic here.